src/parser.rs

```diff
@@ -322,13 +328,28 @@ impl Parser {
         success(encoding, xml)
     }
 
+    fn parse_standalone_declaration<'a>(&self, pm: &mut XmlMaster<'a>, xml: StringPoint<'a>)
+                                        -> XmlProgress<'a, &'a str>
+    {
+        let (xml, _) = try_parse!(xml.expect_space());
+        let (xml, _) = try_parse!(xml.expect_literal("standalone"));
+        let (xml, _) = try_parse!(self.parse_eq(xml));
+        let (xml, standalone) = try_parse!(
+            self.parse_quoted_value(pm, xml, |_, xml, _| xml.consume_yes_no())
```

Comment:
This should probably be something like

```rust
pm.alternate()
    .one(|| literal("yes"))
    .one(|| literal("no"))
    .finish()
```

As it is right now, it could parse as `standalone='yoooooooooooo'` :smiley_cat: